data-egress.tf

```diff
@@ -114,7 +110,7 @@ data "aws_iam_policy_document" "published_non_sensitive_bucket_s3" {
 }
 
 resource "aws_acm_certificate" "data_egress_server" {
-  count                     = local.is_mgmt_env[local.environment] ? 0 : 1
+  count      = local.is_mgmt_env[local.environment] ? 0 : 1
```

Comment:
this doesn't seem right, the previous indent aligned the equals symbols. Could your run `terraform fmt` to confirm?

Reply:
done